File: ignored-test/java-junit/Should_Add_SkipCleanup_Flag_To_Arguments_If_Set.java
import org.junit.Test;
import static org.junit.Assert.assertEquals;
import org.junit.Ignore;

public class ChocolateyPusherTests {

    @Ignore(" ")
    @Test
    public void shouldAddSkipCleanupFlagToArgumentsIfSet() {
        {
            // Given
            ChocolateyPusherFixture fixture = new ChocolateyPusherFixture();
            fixture.getSettings().setSkipCleanup(skipCleanup);
            // When
            var result = fixture.run();
            // Then
            assertEquals(expected, result.getArgs());
        }
    }
}

File: ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/Should_Add_SkipCleanup_Flag_To_Arguments_If_Set.cs
using Xunit;

namespace DefaultNamespace
{
    public class ChocolateyPusherTests
    {
        [Fact(Skip = "")]
        public void Should_Add_SkipCleanup_Flag_To_Arguments_If_Set()
        {
            {
                // Given
                var fixture = new ChocolateyPusherFixture();
                fixture.Settings.SkipCleanup = skipCleanup;
                // When
                var result = fixture.Run();
                // Then
                Assert.Equal(expected, result.Args);
            }
        }
    }
}
